### ISSUE TITLE
Could I kindly consult you on the following questions?

### ISSUE THREAD

**Issue by kongyn-ccc** [Ephemeral182/ECCV24_T3-DiffWeather#12]
Could I kindly consult you on the following questions? I would sincerely appreciate your guidance and clarification.
1、Approximately how long would the training process take when using a 24GB graphics card?
2、The training dataset 18069 I downloaded is stored in a single folder. Do I need to manually split it into corresponding subsets?
3、Is a validation set required during training? If so, should it be sampled from the test dataset or the training dataset?
